README.markdown

```diff
@@ -499,10 +500,10 @@ If the call site must be wrapped, put each parameter on a new line, indented one
 
 ```swift
 let success = reticulateSplines(
-  spline: splines,
-  adjustmentFactor: 1.3,
-  translateConstant: 2,
-  comment: "normalize the display")
+    spline: splines,
```

Comment:
I think the Xcode default of having the first parameter on the same line, then each subsequent parameter on a separate line underneath it, looks better and improves readability.

```
let success = reticulateSplines(spline: splines,
                                                     adjustmentFactor: 1.3,
...`

Reply:
yeah I tend to agree. I have had to fallback to this style though when names are long and things are already nested...Though that is usually an indication things have gone very wrong somewhere along the way.